solr/conf/schema.xml

```diff
@@ -177,11 +160,9 @@
       <analyzer>
         <!-- <tokenizer class="solr.WhitespaceTokenizerFactory"/> -->
         <tokenizer class="solr.StandardTokenizerFactory"/>
-        <filter class="solr.ICUFoldingFilterFactory" />
```

Comment:
Good catch. I was just hacking through the schema to get it working with solr_wrapper/Solr 6. I'm not sure how to implement this to satisfy old and new solr versions. I'll look into it.